openweathermap-ios/openweathermap-ios/MapViewController.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  MapViewController.swift
+//  openweathermap-ios
+//
+
+
+import UIKit
+import MapKit
+
+class MapViewController: UIViewController, CLLocationManagerDelegate {
+
+    @IBOutlet weak var mapView: MKMapView!
+    let locationManager = CLLocationManager()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        mapView.delegate = self
+        locationManager.delegate = self
+        locationManager.desiredAccuracy = kCLLocationAccuracyBest
+        locationManager.requestWhenInUseAuthorization()
+        locationManager.startUpdatingLocation()
+        
+    }
+    
+    
+    func addAnnotation(location: CLLocationCoordinate2D){
+        print("Coordenate center: \(mapView.centerCoordinate)")
+        let circleCenter = MKCircle(center: location, radius: 50)
+        let circleEdge = MKCircle(center: location, radius: 5)
+        
+        mapView.overlays.forEach { (overlay) in
+            if (overlay is MKCircle){
+                mapView.removeOverlay(overlay)
+            }
+        }
+        mapView.addOverlay(circleCenter)
+        mapView.addOverlay(circleEdge)
+    }
+}
+
+extension MapViewController: MKMapViewDelegate{
+    func locationManager(_ manager: CLLocationManager, didChangeAuthorization status: CLAuthorizationStatus) {
+        if status != .authorizedWhenInUse{
+            let alertController = UIAlertController(title: "Permission needed", message: "Without permission the app will not work correctly", preferredStyle: .alert)
```

Comment:
Consider localizing all strings that appear to the user.

Reply:
That would be implementing here with a Switch case .authorizedWhenInUse, .authorizedAlways and .denied.
Would it be this?

Reply:
No, I am referring to Language. It is a good approach to localize the Strings, even supporting only 1 language because it facilitates internationalization later.

Reply:
Content:
- https://medium.com/lean-localization/ios-localization-tutorial-938231f9f881
- https://medium.com/swift-india/localize-your-apps-to-support-multiple-languages-ios-localization-ac7b612dbc58
- https://www.raywenderlich.com/250-internationalizing-your-ios-app-getting-started

Reply:
I understood. I will learn and make the necessary changes. Thanks.

Reply:
If you need any help, please, let me or any other folks know. Don't suffer alone. Ok?

Reply:
Ok. Thank you. You are already helping me a lot with all the feedbacks.

Reply:
> @felipebweber ignore all files `.DS_Store`

How do I ignore the .DS_Store files in git?